src/main/daml/Daml/Finance/Settlement/Batch.daml

```diff
@@ -3,20 +3,17 @@
 
 module Daml.Finance.Settlement.Batch where
 
-import DA.Action (foldrA)
+import DA.Action (foldlA)
```

Comment:
Nice, I had been looking for this function a couple of times and always ended up implementing it from scratch :)